src/test/actions.test.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Tests for all actions the loginserver can perform.
+ * By Mia.
+ * @author mia-pi-git
+ */
+import {Config} from '../config-loader';
+import {strict as assert} from 'assert';
+import {NTBBLadder} from '../ladder';
+import * as utils from './test-utils';
+import SQL from 'sql-template-strings';
+import * as tables from '../tables';
+
+const token = '42354y6dhgfdsretr';
+(Config.testdb ? describe : describe.skip)('Loginserver actions', () => {
```

Comment:
We discussed this on discord

Reply:
Yeah, I just haven't pushed my code for this yet.